website/templates/home.html

```diff
@@ -103,7 +106,7 @@ <h4 style="color: #555; margin-bottom: 20px; font-size: 16px; line-height: 140%"
         </div>
         <div class="command" style="margin-bottom: 20px">
             <div class="label">Run this command:</div>  
-            <div class="text" style="background-color: #fff"><code>parlai display_data -t babi:task1k:1,squad -n 100</code></div>
+            <div class="text" style="background-color: #fff"><code>parlai display_data --task babi:task1k:1,squad -n 100</code></div>
```

Comment:
`-n` --> `--num-examples`?

Reply:
I feel like that one's clear enough from the description